### PR TITLE
[KEYCLOAK-7552] - Documentation on how to enable My Resources in account service

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: authorization_services/topics/service-authorization-uma-account-my-resources.adoc [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [[_service_authorization_my_resources]]
 = Managing Access to Users Resources
 
-Users can manage access to their resources using the {project_name} User Account Service.
+Users can manage access to their resources using the {project_name} User Account Service. To enable this functionality
+you need to first enable `User-Managed Access` to your realm. For that, open the realm settings page
+in {project_name} Administration Console and enable the `User-Managed Access` switch.
```
Comment (matthewhelmke) [inline authorization_services/topics/service-authorization-uma-account-my-resources.adoc]: Users can manage access to their resources using the {project_name} User Account Service. To enable this functionality, you must first enable `User-Managed Access` for your realm. To do so, open the realm settings page in {project_name} Administration Console and enable the `User-Managed Access` switch.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Just a couple of minor changes.

**Comment by pedroigor** [comment]
@stianst can you approve, please ?

**Review by matthewhelmke — APPROVED** [review]

**Comment by pedroigor** [comment]
@matthewhelmke, tks !

**Pull request merged**